src/components/TodoComponents/TodoList.js

```diff
@@ -70,24 +116,18 @@ class ToDoList extends React.Component {
 
         return (
 
-            // use map to map all data from the listOfTasks
-            
-            // note the method binding on the ToDo Form.
-
-            // state is a great way of keeping track of
-            // which objects should change and which should not.
             <div className="ToDoList">
                 <ul>
                 {this.state.tasks.map ( (task,key) =>
-                    <ToDo key={key} task={task} />                    
+                    <ToDo key={key} task={task} toggle = {this.toggle} />                    
                 )}
                 </ul>
                 
                 <ToDoForm
                     addTask = {this.addTask}
                     value = {this.state.task}
                     handleChanges = {this.handleChanges}
-                    onClear = {this.clearCompleted}
+                    clearCompleted = {this.clearCompleted}
```

Comment:
great work using the map and filter array methods you def are becoming very skilled with them nice work